chef/cookbooks/nova/recipes/compute.rb

```diff
@@ -130,19 +130,41 @@ def set_boot_kernel_and_trigger_reboot(flavor='default')
   end
 
   case node[:nova][:libvirt_type]
-    when "kvm"
-      package "kvm"
+    when "kvm", "qemu"
+      if node.platform_version.to_f >= 12.0
+        package "qemu"
+        package "qemu-kvm"
```

Comment:
This one is not needed if libvirt_type is qemu (although it probably does not hurt much?)